apps/fm/index.html

```diff
@@ -25,13 +25,13 @@
     <div id="container" dir="ltr">
       <div id="frequency-bar">
         <div id="frequency-display">
-          <a id="speaker-switch" href="#speaker" data-speaker-on="false"></a>
-          <a id="bookmark-button" href="#bookmark" data-bookmarked="false"></a>
+          <a id="speaker-switch" href="#speaker" data-speaker-on="false" data-l10n-id="speaker-switch" role="button"></a>
+          <a id="bookmark-button" href="#bookmark" data-bookmarked="false" data-l10n-id="bookmark-button" role="button"></a>
           <div id="frequency">0</div>
```

Comment:
Here the markup order is different from the display order. Ideally we would want to fix that with css flex layout. What we want is for markup to be in the same order as the presentation: speaker button , frequency and lastly bookmark. This matters for sequential navigation because at the current state, the focus jumps between buttons and then frequency.

Also I would add role status to the frequency so we get notified when it gets updated.